src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -87,40 +85,40 @@
     private double experience;
 
     /**
-     * Create a CitizenData given an ID
-     * Used as a super-constructor or during loading
+     * Create a CitizenData given an ID.
+     * Used as a super-constructor or during loading.
      *
-     * @param id     ID of the Citizen
-     * @param colony Colony the Citizen belongs to
+     * @param id     ID of the Citizen.
+     * @param colony Colony the Citizen belongs to.
      */
-    public CitizenData(int id, Colony colony)
+    public CitizenData(final int id, final Colony colony)
     {
         this.id = id;
         this.colony = colony;
     }
 
     /**
-     * Creates CitizenData from tag compound
+     * Creates CitizenData from tag compound.
      *
-     * @param compound NBT compound to build from
-     * @param colony   Colony of the citizen
-     * @return CitizenData
+     * @param compound NBT compound to build from.
+     * @param colony   Colony of the citizen.
+     * @return CitizenData.
      */
     @NotNull
-    public static CitizenData createFromNBT(@NotNull NBTTagCompound compound, Colony colony)
+    public static CitizenData createFromNBT(@NotNull final NBTTagCompound compound, final Colony colony)
     {
-        int id = compound.getInteger(TAG_ID);
-        @NotNull CitizenData citizen = new CitizenData(id, colony);
+        final int id = compound.getInteger(TAG_ID);
+        final @NotNull CitizenData citizen = new CitizenData(id, colony);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)